src/ui_fsmenu/addons_packager.cc

```diff
@@ -449,6 +449,7 @@ void AddOnsPackager::clicked_write_changes() {
 	for (const auto& str : addons_with_changes_) {
 		msg = (boost::format(_("%1$s\n· %2$s")) % msg % str.first).str();
 	}
+	msg += _("\n\nBe careful with this, if you have modified addon files yourself!");
```

Comment:
Should be more precise, e.g.
```suggestion
	msg += _("\n\nWarning: If you manually edited any files in this add-on, your changes may be overwritten!");
```